spec/javascripts/collections/story_collection_spec.js

```diff
@@ -1,6 +1,6 @@
 var Story = require('models/story');
 var StoryCollection = require('collections/story_collection');
-
+require('jasmine-ajax');
```

Comment:
We could move it to this file https://github.com/Codeminer42/cm42-central/blob/master/spec/javascripts/index.js#L4 to avoid having to do it again if a different file needs this plugin too.